src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -643,12 +643,11 @@ def test_network_ag_zone(self, resource_group):
             'gateway': 'ag1',
             'ip': 'pubip1'
         })
-        self.cmd('network public-ip create -g {rg} -n {ip} --sku Standard')
-        self.cmd('network application-gateway create -g {rg} -n {gateway} --sku Standard_v2 --min-capacity 2 --max-capacity 4 --zones 1 3 --public-ip-address {ip} --no-wait')
+        self.cmd('network public-ip create -g {rg} -n {ip} --sku Standard -z 1')
+        self.cmd('network application-gateway create -g {rg} -n {gateway} --sku Standard_v2 --min-capacity 2 --max-capacity 4 --zones 1 --public-ip-address {ip} --no-wait')
```

Comment:
Why change the test for list value of `--zones`? Can `1 3` still work for `--zones`?

Reply:
The default behavior about 'zone' changed, and I will support '--zone 1  3' for 'az network public-ip' in another PR. It is new feature for ignite. But before that, I need to bump version.